### DISCUSSION TITLE
GSOC 24: Wanting to contribute to the project: Advancing Bird Detection and classification in Hand-held airborne imagery.

### DISCUSSION THREAD

**Discussion by krish1209** [weecology/retriever#1678] [Q&A] [answered]
I am a final year student pursuing BTech in CSE AI ML, I have experience in Machine learning Frameworks and algorithms, even with Deep learning.
I would like to contribute towards this project as a start towards GSOC 24. 
Could you please provide a guide for the same?
Thank you

**Comment by henrykironde** [comment, marked as answer]
@krish1209 please refer to https://github.com/weecology/retriever/issues/1680
